docs/ensuring-repro/workflows/run-module-gha.md

```diff
@@ -0,0 +1,42 @@
+# Automated module testing
+
+To maintain module functionality over time, we use [GitHub Actions](https://docs.github.com/en/actions) (GHAs) to periodically run each module ("module testing GHA") with the goal of testing that it runs to completion without errors.
+
+!!! info
+    For more information about how we run modules to generate official results and OpenScPCA releases, please see our documentation on the `OpenScPCA-nf` workflow. <!-- openscpca-nf STUB_LINK -->
+
+Module testing GHAs are automatically run under these two circumstances:
+
+- When a pull request is filed with changes to any module files
+    - This GHA will need to pass without errors for [pull requests](../../contributing-to-analyses/pr-review-and-merge/index.md) to be approved
+- All modules are periodically run on a schedule to ensure they continue to pass tests as OpenScPCA grows
```

Comment:
My original motivation to changes here was to make the two bullets parallel, but I'm not totally happy with the result... maybe you can see where I am going?
```suggestion
Module testing GHAs are automatically run in two circumstances:

- When a pull request is filed with changes to any module files
    - This GHA will need to pass without errors for [pull requests](../../contributing-to-analyses/pr-review-and-merge/index.md) to be approved
- On a periodic schedule
  - This ensures that changes in data or other code do not break tests within each module
```

Reply:
I thought your proposed structure was good here!